packages/backend-app-api/src/config/ObservableConfigProxy.ts

```diff
@@ -61,6 +62,12 @@ export class ObservableConfigProxy implements ConfigService {
       },
     };
   }
+  mergeConfig(configData: JsonObject) {
```

Comment:
Let's avoid this addition and merge config data at the `AppConfig` level instead.